scripts/check-mcp-compliance.py

```
@@ -0,0 +1,308 @@
+#!/usr/bin/env python3
+"""MCP Compliance Check - Ensure all components use real MCP, no mocks"""
+
+import os
+import sys
+from pathlib import Path
+from typing import List, Tuple
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)